data/themes/darktable.css

```diff
@@ -2252,6 +2254,11 @@ Details :
   margin: 1.4em;
 }
 
+.dt_thumbnail_rating_6 #thumb_image
+{
+  color: rgba(0,0,0,0.5); /* only the transparency is used to fade the image drawing */
```

Comment:
I would even use .25 for the alpha channel to make it easier to see rejected images.

Reply:
no problem... as said, I've put 0.5 more as an example than anything...
maybe we can wait for @Nilvus review, as I'm quite sure he will have some more css finetuning ?

Reply:
Sure, it was a pure comment and I had added @Nilvus as reviewer for him to have the last word on this :)

Reply:
Just a quick test and maybe something 0.25 and 0.5 would be better. Seems 0.35 is something good. Anyway, I don't have a lot of time this week to test further. Will try to check that on Wednesday, or if not possible, only next week.

Reply:
I would be for 0.35, seems a good compromise. I also think this would be good too to fade other parts of rejected images (ext, group/altered icon and also star when hovering). But all this could be done in a separate PR by me. I so suggested to simply update to 0.35 here and merge if all good for you.